### PR TITLE
Create LauraGdl.json

### PR DESCRIPTION
**Pull request by YessiBarrerra**
Latitud y longitud, close #8125

### CODE REVIEW

**Comment by davidnhz** [comment]
👍

File: _pins/LauraGdl.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: LauraGdl  
+latitude: 19.4794
```
Comment (YessiBarrerra) [inline _pins/LauraGdl.json]: Esta no es mi ubicación

Comment (davidnhz) [inline _pins/LauraGdl.json]: You need to add `---` at the beginning and at the end of the file.

Comment (YessiBarrerra) [inline _pins/LauraGdl.json]: Thanks David 👍

**Comment by crichID** [comment]
@YessiBarrerra your checks are not passing because you have trailing spaces at the end of one or more of you lines of code. If you need help making changes, check out this quick video we made on how to correct common issues in this activity: https://vimeo.com/201011288

**Comment by brianamarie** [comment]
@YessiBarrerra  :wave: Let us know if you want or need any help making these changes. If in a few days the changes still aren't made, one of the GitHub trainers may make the changes for you to keep the repository clean. ✨

**Pull request merged**